src/app/components/Sidebar/index.tsx

```diff
@@ -30,61 +30,93 @@ import { useDispatch, useSelector } from 'react-redux'
+  const size = useContext(ResponsiveContext)
+  const isMediumSize = size === 'medium'
+  const tooltip = (
+    <Box
+      pad={{ vertical: 'small', right: 'medium' }}
+      margin="none"
+      background={props.isActive ? 'background-oasis-blue' : 'component-sidebar'}
+      round={{ size: 'medium', corner: 'right' }}
+    >
+      {props.label}
+    </Box>
+  )
+  return (
+    <Tip content={isMediumSize ? tooltip : undefined} dropProps={{ align: { left: 'right' } }} plain={true}>
+      {props.children}
+    </Tip>
+  )
+}
+
+interface SidebarButtonBaseProps {
+  needsWalletOpen?: boolean
+  icon: JSX.Element
   label: string
 }
 
-export const SidebarButton = ({ secure, icon, label, route, ...rest }: SidebarButtonProps) => {
-  const status = useSelector(selectStatus)
+type SidebarButtonProps = SidebarButtonBaseProps &
+  (
+    | { route: string; newTab?: boolean; onClick?: undefined }
+    | {
+        route?: undefined
+        newTab?: undefined
+        onClick: React.MouseEventHandler<HTMLButtonElement> & React.MouseEventHandler<HTMLAnchorElement>
+      }
+  )
+
+export const SidebarButton = ({
+  needsWalletOpen,
+  icon,
+  label,
+  route,
+  newTab,
+  onClick,
+}: SidebarButtonProps) => {
+  const isWalletOpen = useSelector(selectIsOpen)
   const size = useContext(ResponsiveContext)
   const location = useLocation()
-  const isActive = route && route === location.pathname
+  const isActive = route ? route === location.pathname : false
   const isMediumSize = size === 'medium'
 
-  if (!status && secure) {
+  if (!isWalletOpen && needsWalletOpen) {
     return null
   }
 
-  const tooltip = (
+  const component = (
     <Box
-      pad={{ vertical: 'small', right: 'medium' }}
-      margin="none"
-      background={isActive ? 'background-oasis-blue' : 'component-sidebar'}
-      round={{ size: 'medium', corner: 'right' }}
+      pad={{ vertical: 'small', left: isMediumSize ? 'none' : 'medium' }}
+      background={isActive ? 'background-oasis-blue' : undefined}
+      responsive={false}
+      direction="row"
+      gap="medium"
+      justify={isMediumSize ? 'center' : 'start'}
     >
-      {label}
+      {icon}
+      {!isMediumSize && <Text>{label}</Text>}
     </Box>
   )
 
-  const component = (
-    <Tip content={isMediumSize ? tooltip : undefined} dropProps={{ align: { left: 'right' } }} plain={true}>
-      <Box
-        pad={{ vertical: 'small', left: isMediumSize ? 'none' : 'medium' }}
-        background={isActive ? 'background-oasis-blue' : undefined}
-        responsive={false}
-      >
-        <Button
-          a11yTitle={label}
-          gap="medium"
-          alignSelf={isMediumSize ? 'center' : 'start'}
-          focusIndicator={false}
-          plain
-          icon={icon}
-          label={!isMediumSize ? label : undefined}
-          {...rest}
-        />
-      </Box>
-    </Tip>
-  )
-
   if (route) {
-    return <NavLink to={route}>{component}</NavLink>
```

Comment:
Previously, this created either a button or a link this way:
- button
  ```tsx
  <Tooltip>
    <Button focusIndicator={false} onClick={onClick} />
  </Tooltip>
  ```
  <kbd>Tab</kbd> focused on Button with tooltip, without focus indicator.

- link
  ```tsx
  <NavLink to={route}>
    <Tooltip>
      <Button focusIndicator={false} onClick={undefined} />
    </Tooltip>
  </NavLink>
  ```
  First <kbd>Tab</kbd> focused on NavLink without tooltip, focus indicated with white outline. Next <kbd>Tab</kbd> focused on Button with tooltip, without focus indicator.

---

After:
- button
  ```tsx
  <Tooltip>
    <Button onClick={onClick} />
  </Tooltip>
  ```
  <kbd>Tab</kbd> focuses on Button with tooltip, focus indicated with green outline.

- link
  ```tsx
  <Tooltip>
    <NavLink to={route} />
  </Tooltip>
  ```
  <kbd>Tab</kbd> focuses on NavLink with tooltip, focus indicated with white outline.